api/python/notebooks/comp_bio_normalizing_full_gene_sequencing.ipynb

```diff
@@ -0,0 +1,393 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Normalizing full-length gene sequencing data from the Cell Census\n",
+    "\n",
+    "The Cell Census is a versioned container for the single-cell data hosted at [CELLxGENE Discover](https://cellxgene.cziscience.com/). The Cell Census utilizes [SOMA](https://github.com/single-cell-data/SOMA/blob/main/abstract_specification.md) powered by [TileDB](https://tiledb.com/products/tiledb-embedded) for storing, accessing, and efficiently filtering data.\n",
+    "\n",
+    "This notebook shows you how to fetch full-length gene sequencing data from the Cell Census and normalize it to account for gene lenght.\n",
+    "\n",
+    "\n",
+    "## Contents\n",
+    "\n",
+    "- Opening the census\n",
+    "- Fetching example full-length sequencing data (Smart-Seq2)\n",
+    "- Normalizing expression to account for gene length\n",
+    "- Validation through clustering analysis\n",
+    "\n",
+    "## Opening the census\n",
+    "\n",
+    "First we open the Cell Census, if you are not familiar with the basics of the Cell Census API you should take a look at the notebook \"Learning about the CELLxGENE Cell Census\" in `comp_bio_census_info.ipynb`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import cell_census\n",
+    "import scanpy as sc\n",
+    "\n",
+    "census = cell_census.open_soma()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can learn more about the all of the `cell_census` methods by accessing their corresponding documention via `help()`. For example `help(cell_census.open_soma)`. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Fetching example full-length sequencing data (Smart-Seq)\n",
+    "\n",
+    "Let's get some example data, in this case we'll fetch all cells from a relatively small dataset derived from the Smart-Seq2 technology which performs full-length gene sequencing:\n",
+    "\n",
+    "- Collection: [Tabula Muris Senis](https://cellxgene.cziscience.com/collections/0b9d8a04-bb9d-44da-aa27-705bb65b54eb)\n",
+    "- Dataset: [Liver - A single-cell transcriptomic atlas characterizes ageing tissues in the mouse - Smart-seq2](https://cellxgene.cziscience.com/e/4546e757-34d0-4d17-be06-538318925fcd.cxg/)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "liver_dataset_id = \"524179b0-b406-4723-9c46-293ffa77ca81\"\n",
```

Comment:
I suspect notebook readers will want to know how you determined the dataset_id....   Is that something obvious, or does it warrant elaboration?

Reply:
I added a section for that clarifies this